desktop/src/main/nl/tudelft/ti2806/riverrush/graphics/entity/Animal.java

```diff
@@ -46,6 +46,7 @@ public AnimalActor getActor() {
      */
     public void setActor(final AnimalActor aActor) {
         this.actor = aActor;
+        this.actor.setAnimal(this);
```

Comment:
Alright then it's fine. I was just asking to see if you had considered other possiblities